.github/workflows/_meta.yaml

```diff
@@ -96,9 +96,11 @@ jobs:
 
       - name: Prepare Release Assets
         run: |-
-          find ./ -type f \( -name "*.deb" -o -name "*.zip" \) | while read file; do
-            sha256sum ${file#./} >> ${file%.*}.sha256
+          cd artifact
+          find . -type f \( -name "*.deb" -o -name "*.zip" \) | while read file; do
```

Comment:
This will still have the same problem since `find` will append `./` to the start.

Should just be `for file in *.deb *.zip; do`.

Reply:
Also should likely use `pushd`/`popd` instead of `cd`, but that's down to preference and shouldn't matter in this case.